src/ReadSpeed.cxx

```diff
@@ -235,6 +235,11 @@ Result ReadSpeed::EvalThroughputMT(const Data &d, unsigned nThreads)
    const auto rangesPerFile = MergeClusters(GetClusters(d), maxTasksPerFile);
    clsw.Stop();
 
+   size_t nranges = 0;
+   for (const auto &r : rangesPerFile)
+      nranges += r.size();
```

Comment:
let's [make Sean Parent happy](https://www.youtube.com/watch?v=W2tWOdzgXHA) and use less raw for loops:  

`std::accumulate(rangesPerFile.begin(), rangesPerFile.end(), 0u, [] (size_t s, auto &r) { return s + r.size(); })`